configuration/klippy/beacon_true_zero_correction.py

```
@@ -0,0 +1,430 @@
+				RATOS_TITLE, 
+				'error', 
+				'One or more z values were out of range, maximum retries exceeded.')
+			raise self.gcmd.error('One or more z values were out of range, maximum retries exceeded.')
+		elif isinstance(self._finalize_result, float):
+			if self._finalize_result < -0.2:
+				# Sanity check to reduce the risk of bed damage
+				self.tzc.ratos.console_echo(
+					RATOS_TITLE, 
+					'error', 
+					f'The measured true zero correction {self._finalize_result:.6f} is below the safety limit of -0.2mm._N_This is not expected behaviour.')
+				raise self.gcmd.error(f'Measured correction is below safety limit')
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)